openfisca_us/variables/gov/states/mo/tax/income/credits/mo_property_tax_credit/mo_property_tax_credit_rent_or_own.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_us.model_api import *
+
+class mo_property_tax_credit_rent_or_own(Variable):
```

Comment:
Make this a `rents` bool variable in the household expenses folder (in general two-item enums can usually be bools)

We can't break down parameters by bools, so the parameter will need to be split by rents / non-rents